app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -142,7 +142,14 @@ class HomePresenter(
                 view?.showRefreshing(true)
                 userInteractor.loadAllTokensDataIfEmpty()
                 val solTokens = userInteractor.loadUserTokensAndUpdateLocal(tokenKeyProvider.publicKey.toPublicKey())
-                async { userInteractor.loadUserRates(solTokens) }
+                async {
+                    try {
+                        userInteractor.loadUserRates(solTokens)
+                    } catch (t: Throwable) {
+                        Timber.i(t, "Error on loading user rates")
```

Comment:
i or e?

Reply:
Definitely `i` as it's a non-critical error